fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractWholeSystemSearchTest.java

```diff
@@ -70,16 +72,16 @@ public void testSearchAllUsingId() throws Exception {
         assertTrue(resources.size() > 0);
     }
     
-    @Test
+    @Test(singleThreaded = true)
```

Comment:
does it need to be singleThreaded?  maybe leave comment to explain why?

Reply:
No, I thought I had removed that after my tests.  thanks for identifying, intend to remove.